Druid-Tests/SimpleDruidTestRTLCompiler.class.st

```diff
@@ -3785,19 +3785,13 @@ SimpleDruidTestRTLCompiler >> gen_primitiveLessThanValue [
 SimpleDruidTestRTLCompiler >> gen_primitiveLoopIncrementing [
 	"AutoGenerated by Druid"
 
-	| jump1 s2 currentBlock b16 jump2 |
+	| s6 s2 currentBlock s7 b16 s3 |
 	s2 := 10.
-	self MoveCq: s2 R: ClassReg.
+	s3 := s2.
 	b16 := self Label.
-	self MoveR: ClassReg R: SendNumArgsReg.
-	self AddCq: 1 R: SendNumArgsReg.
-	self CmpCq: 15 R: SendNumArgsReg.
-	jump1 := self JumpGreaterOrEqual: 0.
-	self MoveR: SendNumArgsReg R: ClassReg.
-	jump2 := self Jump: b16.
-	currentBlock := self Label.
-	jump1 jmpTarget: currentBlock.
-	self MoveR: SendNumArgsReg R: ReceiverResultReg.
+	s6 := s3 + 1.
+	[ ((s6<15)) ] whileTrue: [ s3 := s6 ].
+	self MoveCq: s6 R: ReceiverResultReg.
```

Comment:
I don't think that this is the fault of this PR.
I ran the tests locally so this code was regenerated recently...